cmd/krel/cmd/gcbmgr.go

```diff
@@ -126,6 +127,12 @@ func init() {
 			release.VersionTypeCILatest,
 		),
 	)
+	gcbmgrCmd.PersistentFlags().BoolVar(
+		&gcbmgrOpts.UseKrel,
+		"krel",
```

Comment:
Nit: `krel gcbmgr --krel` has a bit of a stutter. I know it's intended to be a temporary flag, but could we use something that's a bit more specific?
Having trouble with naming inspiration, but maybe:
- `--use-krel`
- `--no-shell|-bash|-anago`
- `--go-only`

Reply:
Decided to go with `--no-anago`. :+1: